one_fm/api/utils.py

```diff
@@ -26,6 +26,13 @@ def response(code, title, msg, data=None):
 def get_reports_to_employee_name(employee):
     reports_to = frappe.db.get_value('Employee', employee, 'reports_to')
     if reports_to: return reports_to
+    
+    if not reports_to:
+        site = frappe.db.get_value('Employee', employee, 'site')
+        if site:
+            reports_to = frappe.db.get_value('Operations Site', site, 'account_supervisor')
+            return reports_to
```

Comment:
it can be changed to

`return frappe.db.get_value('Operations Site', site, 'account_supervisor')`

Or

```

  reports_to = frappe.db.get_value('Operations Site', site, 'account_supervisor')

  if reports_to:

     return reports_to

```

Reply:
Done!